kafka/consumer/main.go

```diff
@@ -251,11 +261,22 @@ func (cgh *consumerGroupHandler) ConsumeClaim(session sarama.ConsumerGroupSessio
 	}
 }
 
-func inClusterKafkaConfig() (kafkaConfig *sarama.Config, err error) {
-	kafkaConfig = sarama.NewConfig()
-	kafkaConfig.Consumer.Offsets.Initial = sarama.OffsetNewest
-	kafkaConfig.Version = sarama.V0_10_2_1
-	return kafkaConfig, nil
+func getConfig() *sarama.Config {
+	config := sarama.NewConfig()
+	config.Net.DialTimeout = 10 * time.Second
+
+	config.Net.SASL.Enable = true
+	config.Net.SASL.User = "$ConnectionString"
```

Comment:
This connection string contains credentials.  We should look at releasing this from AKV instead of storing in env.